superset/assets/src/chart/Chart.jsx

```diff
@@ -88,12 +88,21 @@ class Chart extends React.PureComponent {
     });
   }
 
+  renderStackTraceMessage() {
+    const { chartAlert, chartStackTrace } = this.props;
+    return (
+      <StackTraceMessage
+        message={chartAlert}
+        link={queryResponse ? queryResponse.link : null}
+        stackTrace={chartStackTrace}
+      />);
```

Comment:
nit -- missing new line before `);`

Reply:
I'm ok either way personally but let's get the linter to complain if we do care on way or the other. Leaving as is for now.